main.go

```diff
@@ -212,17 +219,32 @@ func (m *MetricMap) Register() error {
 	return nil
 }
 
-func (m *MetricMap) UpdateMetricValue(metricName, timespan, repo string, value float64) error {
+func (m *MetricMap) UpdateMetricValue(metricName, timespan, repo string, value float64, labels []MetricLabel) error {
+
+	stringMap := make(map[string]string)
```

Comment:
Could we rename this variable to something about what in contains instead of its type? `stringMap` is inferred by the type of the value.

Maybe rename the incoming `labels` parameter to `staticLabels` and rename `stringMap` to `labels`. This way it is more clear what the operation is in the next few lines, ie. add static labels to the metric labels.